spec.bs

```diff
@@ -4306,8 +4306,9 @@ runs; this method exists as an abstraction to help add that.
 
 <div algorithm>
 To <dfn>commit private aggregation contributions</dfn> given an [=Private Aggregation
-contributions=] |onEventMap|, a [=reporting bid key=] |bidKey|, and a [=reporting context=]
+contributions=] or null |onEventMap|, a [=reporting bid key=] |bidKey|, and a [=reporting context=]

```

Comment:
when is |onEventMap| null? I briefly checked the callers of this algorithm, and didn't find when it's null

Reply:
The error case you pointed out in evaluate reporting script... but I changed it to return a new value, now that I remembered I have a typedef for it, so this is gone now.